TsRandomizer/Randomisation/ItemLocationMap.cs

```diff
@@ -243,7 +243,7 @@ void AddPresentItemLocations()
 			Add(new ItemKey(2, 56, 840, 192), "Backer room chest 2", ItemProvider.Get(EInventoryUseItemType.EssenceCrystal), UpperLeftLibrary);
 			Add(new ItemKey(2, 56, 1064, 192), "Backer room chest 1", ItemProvider.Get(EInventoryUseItemType.MagicMarbles), UpperLeftLibrary);
 			areaName = "Varndagroth Tower Left";
-			Add(new ItemKey(2, 34, 232, 1200), "Elevator Key not required", ItemProvider.Get(EInventoryUseItemType.FiligreeTea), MidLibrary); //Default item is Jerky, got replaced by FiligreeTea
+			Add(new ItemKey(2, 34, 232, 1200), "Ground Floor", ItemProvider.Get(EInventoryUseItemType.FiligreeTea), MidLibrary); //Default item is Jerky, got replaced by FiligreeTea //more descriptive
```

Comment:
This isn't on the ground floor, it's in the middle of the left tower. You just don't need the Elevator key to get it